sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -255,7 +255,7 @@ export class ManagedIdentityCredential implements TokenCredential {
               logger.info(`token = ${resultToken.token}`);
               return {
                 accessToken: resultToken?.token,
-                expiresInSeconds: resultToken?.expiresOnTimestamp,
+                expiresInSeconds: resultToken?.expiresOnTimestamp / 1000,
```

Comment:
if this is `undefined` won't this return NaN? Also what if it's not divisible by `1000`?

Perhaps something more robust like
```
expiresInSeconds: resultToken?.expiresOnTimestamp ? Math.ceil(resultToken?.expiresOnTimestamp / 1000) : undefined
```